server/db/models/orders.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+var Sequelize = require('sequelize');
+
+module.exports = function (db) {
+
+    db.define('order', {
+        address: {
+            type: Sequelize.TEXT
+        },
+        status: {
+            type: Sequelize.ENUM('pending', 'active', 'fulfilled', 'returned')
+        },
+        shipDate: {
+            type: Sequelize.DATE
+        }
+    }, {
+        instanceMethods: {
+            getTotal: function() {
+                return this.getOrderDetails()
+                    .then(function(orderDetailObjects) {
+                        return orderDetailObjects.reduce(function(a, b) {
+                            return a.subtotal + b.subtotal;
+                        }, { subtotal: 0 });
+                    })
+            }
+        }
+    });
+
+    db.define('orderDetail', {
```

Comment:
This handles rentals well, but buys should probably not rely on the 'rentalDays' property/column.  We should consider a buy vs. rental flag & include it in the subtotal logic.